tests/unit/source_code/test_dbfsqueries.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from databricks.labs.ucx.source_code.base import Deprecation
+from databricks.labs.ucx.source_code.dbfsqueries import FromDbfsFolder
+
+
+@pytest.mark.parametrize(
+    "query",
+    [
+        "SELECT * FROM old.things LEFT JOIN hive_metastore.other.matters USING (x) WHERE state > 1 LIMIT 10",
+        "SELECT * FROM json.'s3a://abc/d/e/f'",
+        # Make sure non-sql doesn't just fail
+        "print('hello')",
+        "",
+    ],
+)
+def test_non_dbfs_trigger_nothing(query):
+    ftf = FromDbfsFolder()
+    assert not list(ftf.lint(query))
+
+
+@pytest.mark.parametrize(
+    "query, table",
+    [
+        ('SELECT * FROM parquet.`dbfs:/...` LIMIT 10', "dbfs:/..."),
+        ("SELECT * FROM delta.`/mnt/...` WHERE foo > 6", "/mnt/..."),
```

Comment:
missing test case for non-dbfs raw access: SELECT * FROM delta.`s3a://foo/bar`

Reply:
Tha is non-dbfs, so I should teset that that statement does not raise an Advisory, in the test above: 
```python
def test_non_dbfs_trigger_nothing(query):
```
?